pkg/slayers/path/scion/decoded.go

```diff
@@ -140,7 +140,7 @@ func (s *Decoded) SerializeTo(b []byte /*@, ghost ubuf []byte @*/) (r error) {
 	//@ sl.SplitByIndex_Bytes(b, 0, len(b), MetaLen, writePerm)
 	//@ sl.Reslice_Bytes(b, 0, MetaLen, writePerm)
 	//@ unfold acc(s.Base.Mem(), R1)
-	if err := s.PathMeta.SerializeTo(b); err != nil {
+	if err := s.PathMeta.SerializeTo(b[:MetaLen]); err != nil {
```

Comment:
Can you please double check that the upstream version also performs this subslicing operation?

Reply:
No it actually does not. Thank you for spotting this!
(For reference, it can be seen here: https://github.com/scionproto/scion/blob/master/pkg/slayers/path/scion/decoded.go#L67)

I have therefore reverted this change and removed the gobra ghost function calls that were previously there to verify the slicing operation. Please let me know if this is correct. 

(It verifies locally.)